setup.py

```diff
@@ -28,10 +28,10 @@
         ]
     },
     install_requires=[
-        "transformers>=4.10.0",
+        "transformers>=4.20.1",
         "datasets<2.0.0",

```

Comment:
@wulu473 the release of `datasets` with the `tqdm` changes is `2.0.0`. I want to be cautious, but I guess we need to demand `>=2.0.0` now. Thoughts?